pymc3/variational/updates.py

```diff
@@ -209,6 +226,7 @@ def sgd(loss_or_grads, params, learning_rate=1e-3):
 
 
 class Sgd(Optimizer):
+    __doc__ = _make_doc(sgd.__doc__)
```

Comment:
Another option would be to move this into the base class `__new__` and do `__doc__ = _make_doc(self._opt.__doc__)`.